configs/helm/templates/grafanadashboard.yaml

```
@@ -8,6 +8,7 @@ kind: GrafanaDashboard
 metadata:
   name: {{ $path | base | trimSuffix ".json" }}
 spec:
+  allowCrossNamespaceImport: {{ .Values.dashboards.allowCrossNamespaceImport }}
```

Is this change backward compatible? 
Else we can add a default to not break current configs: 
```suggestion
  allowCrossNamespaceImport: {{ .Values.dashboards.allowCrossNamespaceImport | default "true" }}
```

---

We are indeed changing the default deployment behaviour, but there will be no regression for currently deployments.
It may unblock new users by allow a less restrictive approach.

It should be the default value as it's seems to be the standard on other open-source projects and it's the expect default behaviour on majority of the deployments.